README.md

```diff
@@ -72,21 +78,39 @@ at runtime; XML is processed when the server starts up for actual world data
    TCP/6379, can be accessed from the server running Hybrasyl; you may need to
    grant access or open ports.
 
-2. Do _one_ of the following. Either:
-   * Run the included powershell script (`Prep.ps1`) included in the `examples` directory 
-   to create the Hybrasyl data directories. Hybrasyl's data is currently located at ```%MYDOCUMENTS\Hybrasyl\world```, normally found at `C:\Users\<yourusername>\Documents\world`.
-
-	* Copy the [example XML and scripting data](https://github.com/hybrasyl/server/tree/master/examples/XML)
-   (including subdirectories) from the examples directory into the `world\xml` directory. This 
-   will populate the world with enough to login as a user,wander around, and test functionality.
-   
-   **_or_**
-   
-   * Unzip the included `examples.zip` into your Hybrasyl folder.
- 
-3. Examine the Hybrasyl configuration in the Hybrasyl data directory, `config.xml`. In particular, 
-   you will want to add the name of your character to `<Privileged>`, which will allow 
-   them to use any slash command.  
+2. Create and populate your base directory.
+
+   On Windows, this is `%userprofile%\documents\Hybrasyl`. On GNU/Linux
+   or OSX this is `~/Hybrasyl` for whatever user is running the server.
+   Take a look at the
+   [community-maintained database](https://github.com/shadowoffice/HybraDB)
+   for XML and scripting. This has more than enough XML and scripts to
+   get you started. You can put the contents of that repository
+   directly into your Hybrasyl data directory and start the server. You
+   may wish to modify the default `config.xml`.
+
+3. Update your configuration.
+
+   Examine the Hybrasyl configuration in the Hybrasyl data directory, `config.xml`. In particular,
+   you will want to add the name of your character to `<Privileged>`, which will allow
+   them to use any slash command.
+
+4. Install and run Hybrasyl (see _Running Hybrasyl_ below).
+
+## Running Hybrasyl
+
+Hybrasyl Server is .NET Core, which means it can be run on a variety of platfoms (Windows, GNU/Linux, OSX).
+
+A `systemd` unit file [is provided](./contrib/hybrasyl.unit) to start the server on Ubuntu 18.04+. In any case,
+[download the latest release](https://github.com/hybrasyl/server/releases) for your platform. This can be unpacked
+into `/srv/hybrasyl` on GNU/Linux or a directory of your choosing on Windows.
+
+Once downloaded, either run the server directly (`Hybrasyl.exe` or
+`Hybrasyl` on GNU/Linux) or, if you’re running on GNU/Linux or a WSL
+distribution that uses systemd, install the unit file in
+`/etc/systemd/system/hybrasyl.service` and start Hybrasyl:
+
+`service hybrasyl start`
```

Comment:
may want to include `dotnet run` commands for running it as a non-service on osx/linux

Reply:
My intent is to publish artifacts so you don't need dotnet run; the build process builds as ELF and can be run from command line normally (literally just `./Hybrasyl`)

Reply:
sure, but in the cast of debug/test on osx/linux the run commands will help

Reply:
👍 - will add